iModelCore/ECDb/Tests/NonPublished/ECSqlStatementTests.cpp

```diff
@@ -11466,4 +11466,561 @@ TEST_F(ECSqlStatementTestFixture, SelectAnySomeAll)
         ASSERT_EQ(expected, GetHelper().ExecuteSelectECSql(ecsql));
         }
     }
+
+//---------------------------------------------------------------------------------------
+// @bsimethod
+//+---------------+---------------+---------------+---------------+---------------+------
+TEST_F(ECSqlStatementTestFixture, SQLiteWindowFunctions)

```

Comment:
It's a bit confusing how the tests are spread across 4 different tests with no clear separation (some of these queries in this test seem to better fit into the WindowPartitionClause test)
Also the schema and the test data (except for 3 entries in WindowPartitionClause) are identical.
I believe this deserves a dedicated file, you can setup and insert the data in a single setup method, since all the tests only run queries. And then each individual query can be a separate test. That's the better practice since a failing build will immediately tell which individual test failed instead of "SQLiteWindowFunctions failed"

Reply:
fixed?